tests/pipeline-all-amps/Readme.md

```diff
@@ -0,0 +1,11 @@
+# ACS Pipeline-all-amps Repo & Share Docker Image
```

Comment:
Replace with:

# ACS Pipeline-all-amps Repo & Share Docker Images

The [pipeline-all-amps](../pipeline-all-amps) module pushes repo and share images with extra amps applied on top for use by the CI/CD single pipeline process.

## Travis job to test and push docker images

acs-packaging repo [travis](../../.travis.yml) job `"Pipeline All AMPs tests and push docker images"` publishes internally the new docker images for repo and share with travis build number `Example: alfresco/alfresco-pipeline-all-amps-repo:3102:7.0.0`. The images has been tested using [docker-compose](../tests/environment/docker-compose-pipeline-all-amps.yml) deployment.

Repo and share images are pushed to [Quay docker repo](https://quay.io/repository/alfresco/alfresco-pipeline-all-amps-repo?tab=info) and [Quay docker share](https://quay.io/repository/alfresco/alfresco-pipeline-all-amps-share?tab=info)

Reply:
Done